setup.py

```diff
@@ -18,9 +18,9 @@ def readme():
     packages=find_packages(),
     license="Apache 2.0",
     install_requires=[
-        "numpy~=1.15",
+        "numpy>=1.15.0,<1.21.0",
```

Comment:
I don't think we need to pin numpy here since TensorFlow tends to have a constaint on the numpy version used anyway.

Reply:
I'm not sure, I think we do need to declare it as a dependency because it is used in our code. Like, if somebody installed Zoo in a fresh environment, without Numpy in our deps they might see an import error for Numpy, when what they actually are missing is TF.

Reply:
Oh yeah, I just meant we can probably keep it as `numpy~=1.15` here, but I am also OK with limiting it to `numpy>=1.15.0,<1.21.0` if you prefer.

Reply:
Yeah I'd probably rather stick with this if that's okay, just for consistency if nothing else.